iceberg/iceberg-handler/src/main/java/org/apache/iceberg/mr/hive/HiveIcebergMetaHook.java

```diff
@@ -337,6 +342,23 @@ public void preAlterTable(org.apache.hadoop.hive.metastore.api.Table hmsTable, E
     }
   }
 
+  @NotNull
+  private static Optional<Long> getTxnId() {
+    Optional<Long> txnId;
+    txnId = Optional.empty();
+
+    SessionState sessionState = SessionState.get();
+
+    if (sessionState != null) {
+      HiveTxnManager txnMgr = sessionState.getTxnMgr();
+      if (txnMgr != null) {
+        txnId = Optional.of(txnMgr.getCurrentTxnId());
+      }
+    }
```

Comment:
this part is dupe in
```
HiveTableOperations.java
```
and this in static method, can we keep this logic at one place & use it in the another?

Reply:
As we have discussed with @deniskuzZ , I reverted the 'Pass txnid to MetaStoreLock' commit

Reply:
Fixed.